src/ui/public/test_harness/test_harness.js

```diff
@@ -1,62 +1,29 @@
-/* global mocha */
-
 // chrome expects to be loaded first, let it get its way
 import chrome from 'ui/chrome';
 
-import Nonsense from 'Nonsense';
```

Comment:
I couldn't find any other usages of Nonsense.  Think it's safe to remove the dependency from package.json?